src/components/Notifications/EventLog/EventLogToolbar.tsx

```diff
@@ -39,44 +38,19 @@ interface EventLogToolbarProps extends OuiaComponentProps {
 }
 
 export const EventLogToolbar: React.FunctionComponent<EventLogToolbarProps> = (props) => {
-
-    const filterMetadata = React.useMemo<ColumnsMetada<typeof EventLogFilterColumn>>(() => {
-        const bundleOptions = props.bundleOptions;
-        const applicationOptions = props.applicationOptions;
-        return {
+    const filterMetadata = React.useMemo<ColumnsMetada<any>>(() => {
+        const metaData: ColumnsMetada<any> = {
```

Comment:
Instead of putting into variable, just return directly from the memo since it is just constant